tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -824,5 +830,39 @@ int main(int Ac, char **Av) {
                 << ", size in bytes = " << SpecConst.Size
                 << ", type = " << SpecConst.Type << "\n";
   }
+
+  if (SPIRVPrintReport) {
+    std::ifstream IFS(InputFile, std::ios::binary);
+    int ErrCode = 0;
+    std::optional<SPIRV::SPIRVModuleReport> BinReport =
+        SPIRV::getSpirvReport(IFS, ErrCode);
+    if (!BinReport) {
+      std::cout << "Invalid SPIR-V binary, error code is " << ErrCode << "\n";
```

Comment:
Can we add a test for this error reporting, if possible?

Reply:
Sure, thank you.

Reply:
Added in a new commit a test file to test negative condition.